virttest/qemu_devices/qdevices.py

```diff
@@ -1277,6 +1278,8 @@ def hotplug_qmp(self):
         convert_size = utils_misc.normalize_data_size
         args = (params["size"], "B", 1024)
         params["size"] = int(float(convert_size(*args)))
+        if params.get("prealloc-threads"):
+            params["prealloc-threads"] = int(params["prealloc-threads"])
```

Comment:
```suggestion
            params["prealloc-threads"] = params.get_numeric("prealloc-threads")
```

BTW, does it only supports integers? Will there be version compatibility issues?

Reply:
Yes, it's integer only since it's the number of threads. It gets supported since qemu5.0, I don't think there will be compatibility issue. Thanks.

Reply:
Why do not use `params.get_numeric` as @PaulYuuu  suggested before ?

Reply:
params is a dict after L1276, it doesn't have this function.

Reply:
Ah~  got it, thanks~